.github/ISSUE_TEMPLATE/pluto_update_minor.md

```diff
@@ -15,29 +15,29 @@ PLUTO uses version numbering YYvMAJOR.MINOR
 2. MINOR version for using the latest versions of particular input data
 
 Make sure version name is updated in this file
-- [ ] <https://github.com/NYCPlanning/db-pluto/blob/main/pluto_build/version.env>
+- [ ] <https://github.com/NYCPlanning/data-engineering/blob/main/products/db-pluto/pluto_build/version.env>
 
 In that file, the following are also specified. These versions are to be help constant with what was used in the last major pluto update.
 They are not used by the main/major data loading script, so they are only updated at the first minor release of each major release.
 For example, say 22v1.4 is the last release. 22v2 is to be built. These args are not used in the build process, so they do not need to be updated.
 When it's time to build 22v2.1, these must be updated to match [the versions used in 22v2](https://nyc3.digitaloceanspaces.com/edm-publishing/db-pluto/22v2/latest/output/source_data_versions.csv) as they were last used for 22v1 minor versions. Finally, when 22v2.2 is being built, they don't need to be updated since they've
 already been updated to be set to 22v2 versions
 
-- [ ] DOF_WEEKLY_DATA_VERSION
-- [ ] DOF_CAMA_DATA_VERSION
+- [ ] DOF_WEEKLY_DATA_VERSION - temporarily split out to pts, numbldgs since pts temporarily fell slightly out of date. Can be rejoined to one var in minor builds of 23v3
+- [ ] PLUTO_INPUT_CAMA_VERSION
 
 - [ ] GEOSUPPORT_VERSION
-- [ ] FEMA_FIRPS_VERSION
-- [ ] DOITT_DATA_VERSION
-- [ ] DOF_DATA_VERSION
+- [ ] FEMA_FIRMS_VERSION
+- [ ] DOITT_ZIPCODEBOUNDARIES_VERSION
+- [ ] DOF_CONDO_SHORELINE_VERSION
```

Comment:
Just renaming things to align with their names in edm-recipes. Same thing done in version.env